Dockerfile

```diff
@@ -18,7 +18,11 @@ COPY src/backend/setup.* /builder/
 # Needs to be kept before the `pip install`
 RUN apt-get update && \
     apt-get install -y \
-    libxmlsec1-dev && \
+        pkg-config \
+        gcc \
```

Comment:
Isn't leaving gcc installed a security issue ?

Reply:
it is only installed in the `back-builder` layer. It is not present anymore in the last one. You can check by running

```
$ docker run --rm -it marsha:latest gcc
/usr/local/bin/entrypoint: 32: exec: gcc: not found
```